package-lock.json

```
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "0.70.1",
+      "version": "0.70.11",
```

we have the project version `0.70.11` and packages version `0.70.1`

```suggestion
      "version": "0.70.1",
```